lib/GitHubFileCopier.js

```diff
@@ -0,0 +1,90 @@
+'use strict'; // eslint-disable-line
+
+const requestPromise = require('request-promise');
+
+class GitHubFileCopier {
+  constructor(options) {
+    this.githubOrg = options.githubOrg;
+    this.githubUser = options.githubUser;
+    this.githubAccessToken = options.githubAccessToken;
+    this.targetFile = options.targetFile;
+    this.destinationRepo = options.destinationRepo;
+    this.destinationPath = options.destinationPath;
+    this.destinationBranch = options.destinationBranch;
+
+    this.rp = requestPromise.defaults({
+      baseUrl: 'https://api.github.com/repos/',
+      auth: {
+        user: this.githubUser,
+        pass: this.githubAccessToken,
+        sendImmediately: true,
+      },
+      headers: {
+        'User-Agent': this.githubOrg,
+        Accept: 'application/vnd.github.v3+json',
+      },
+    });
+  }
+
+  wasTargetUpdated(payload) {
+    if (!payload.ref) {
+      return false;
+    }
+
+    if (payload.repository.full_name.indexOf(this.githubOrg) !== 0 ||
```

Comment:
I wonder if it makes sense to do `indexOf(this.githubOrg + '/')` just to be safe/explicit?